package.json

```diff
@@ -4,7 +4,7 @@
   "homepage": "https://github.com/terser-js/terser",
   "author": "Mihai Bazon <mihai.bazon@gmail.com> (http://lisperator.net/)",
   "license": "BSD-2-Clause",
-  "version": "4.1.4",
+  "version": "4.1.5",
```

Comment:
I pretty sure this should not be changed

Reply:
Yeah, I handle version bumps :)